conda-env.yml

```diff
@@ -0,0 +1,17 @@
+name: motile-tracker-conda
+channels:
+  - conda-forge
+  - funkelab
+  - gurobi
+dependencies:
+  - python=3.11.*
+  - pip
+  - gcg
+  - papilo
+  - scip
+  - soplex
+  - zimpl
+  - ilpy=0.4.0
```

Comment:
Ideally we would not pin ilpy to 0.4.0, since 0.4.1 has some bugfixes that I have needed. If the above doesn't help, and you can identify a problem with 0.4.1, we should open an issue on ilpy and I ask our collaborator to fix it.